gravitee-apim-portal-webui-next/src/app/app.routes.ts

```diff
@@ -41,10 +42,12 @@ export const routes: Routes = [
           {
             path: 'details',
             component: ApiTabDetailsComponent,
+            data: { breadcrumb: 'Details' },
```

Comment:
Does this support translations? If not, we need to find another solution....

Reply:
For now, we can have a very simple component by hand (without the xng-breadcrumb package) that shows "Catalog > {api.name}"... The Catalog html can use the same i18n as the Catalog button.

Reply:
Done